src/nemo-properties-window.c

```diff
@@ -2148,6 +2164,21 @@ directory_contents_value_field_update (NemoPropertiesWindow *window)
 			    text);
 	g_free (text);
 
+	/* Update the file size field with the selection or directory's total file size. */
+	if (is_computer_directory (file) ||
+	    is_network_directory (file)) {
```

Comment:
You're using `file` here which isn't initialized except inside of a loop at the beginning of `directory_contents_value_field_update()` - so it seems like you'll end up always evaluating the last file of the selection.

You could maybe check this for each file inside the loop, and if any are the pseudo file-type you can act on it here. Or maybe you only need to check selection's parent directory?

Reply:
Interesting.. In case of a multi-selection, the var `total_size` is calculated in the first `for` loop, which is then used to display the file size. In case a single directory is selected, the for loop initializes `file` with this exact directory.

So as long as you don't have the special `computer:///` or `network:///` "directories" in a multi-selection, this should even work as intended :smile:

Reply:
@mtwebster I changed the code a bit to make it more clear